types/puppeteer/v2/index.d.ts

```diff
@@ -293,6 +293,19 @@ export interface MousePressOptions {
   clickCount?: number;
 }
 
+export interface MouseWheelOptions {
+	/**
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/puppeteer/v2/index.d.ts;
  Line: 297;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.